content/zpages.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Z-Pages"
+type = "leftnav"
++++
+
+OpenCensus provides in-process web pages that displays
+collected data from the process. These pages are called z-pages
+and they are useful to see collected data from a specific process
+without having to depend on any metric collection or
+distributed tracing backend.
+
+Z-Pages can be useful during the development time or when
```

Comment:
[TraceConfigz](https://github.com/census-instrumentation/opencensus-java/tree/master/contrib/zpages#view-and-update-tracing-configuration-on-traceconfigz-page) can also be used to update active params (e.g the ones in https://github.com/census-instrumentation/opencensus-specs/pull/139).

Reply:
Go doesn't support config push at this point, should we leave this out until we have feature parity not overpromise to the user?

Reply:
Yep, I think that makes more sense.